docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -32,12 +32,13 @@ exclude fields. For more information about field selection, refer to the
 
 //tag::classification-algorithms[]
 {classanalysis-cap} uses an ensemble algorithm that is a type of boosting called 
-boosted tree regression model which combines multiple weak models into a 
+extreme gradient boost (XGboost) which combines multiple weak models into a 
 composite one. It uses decision trees to learn to predict the probability that a 
-data point belongs to a certain class. A sequence of decision trees are trained 
-and every decision tree learns from the mistakes of the previous one. Every tree 
-is an iteration of the last one, hence it improves the decision made by the 
-previous tree.
+data point belongs to a certain class. XGboost trains a sequence of decision 
+trees and avery decision tree learns from the mistakes of the previous one. 
+Every tree is an iteration of the last one, hence it improves the decision made 
+by the previous tree. The classification algorithm is optimized for a loss 
```

Comment:
Thus, in each iteration, we add trees that improve the decision quality of the combined decision forest.

Reply:
The classification algorithm **optimizes for** ...